apps/web/src/pages/SearchRequests/ViewSearchRequestPage/components/UpdateSearchRequest.tsx

```diff
@@ -78,112 +86,170 @@ export const UpdateSearchRequestForm = ({
       });
   };
 
-  const handleStatusChangeToDone: SubmitHandler<FormValues> = async (
+  const handleSaveStatus: SubmitHandler<FormValues> = async (
```

Comment:
I deleted `handleStatusChangeToDone` and added in `handleSaveStatus`, the git difference decided to make it look rough